account_analytic_distribution/__init__.py

```diff
@@ -1,5 +1,5 @@
-# -*- coding: utf-8 -*-
-# Copyright 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
```

Comment:
PLease don't remove credits. I would not even claim copyright on migrations of a mechanical nature. Adding your name to the contributor's list is enough.

Reply:
Wasn't my intention at all. Sorry. It's the default file header for newly created python files on my system. I was just focusing on functionality. ;-)

Reply:
The sorry broke my heart.